### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by tchx84**

### CODE REVIEW

**Comment by tchx84** [comment]
@Arxcis is this correct?

File: src/widgets/aboutDialog.ui [hunk 1]
```diff
@@ -33,7 +33,8 @@ Mufeed Ali
 Leorize
 Christopher Davis
 Lctrs
-Will Thompson</property>
+Will Thompson
+Jonas Solsvik</property>
```
Comment (Arxcis) [inline src/widgets/aboutDialog.ui]: Thank you very much @tchx84 🙏 It is an honor to be included in this list 😁

Comment (Arxcis) [reply src/widgets/aboutDialog.ui]: I would prefer if you add my middle name as well 👍 
```suggestion
Jonas Johan Solsvik</property>
```

Comment (tchx84) [reply src/widgets/aboutDialog.ui]: Of course!

**Pull request merged**